### ISSUE TITLE
About the log_likelihood of PCAE.

### ISSUE THREAD

**Issue by ghost** [phanideepgampa/stacked-capsule-networks#12]
Hello, I don't know what's means of this line(line:72 of model.py):
`log_likelihood = torch.sum(pre_ll,dim=1).log().sum(-1).sum(-1).mean() #scalar loss`
This is not match with the formula in the paper, Could you tell me some detail about why this way worked?

**Comment by phanideepgampa** [comment]
Hello @0xdream,

The image likelihood is given in **equation no 10**  in the PCAE section of the paper. 
![image](https://user-images.githubusercontent.com/4285772/96268921-45551d00-0fe7-11eb-8d36-bdd2a310d750.png)

We try to maximise the log-likelihood of the above equation. When we apply the logarithm to the equation,  the product converts to summation which would become _sum__{i,j}_ (log(_sum_m_ probability term)). The mean is over the batch of images.

I hope my explanation is clear. Feel free to reach out if not clear :)
